square_root.py

```diff
@@ -0,0 +1,13 @@
+print("Let me find the square root of your number!")
+x =input("Please input a positive number: ")
+float_x = float(x)
+count = 1
+y = float_x/2
+
+while pow(y, 2) >= (float_x + .01 or float_x - .01):
```

Comment:
This line isn't doing what you might imagine. `(float_x + .01 or float_x - .01)` is evaluated first, since it's in parentheses. If `float_x` is 9.0, then it runs `9.01 or 8.99`. The `or` returns the first true value, 9.01 -- which is true in some metaphysical sense we can discuss in class -- so you're comparing `pow(y, 2)` to 9.01.

This works because you're starting with a big number and going down, but the actual logic isn't what you were going for.